### PR TITLE
Cleaned up CodeQL and Updated CI Process

### PR DESCRIPTION
**Pull request by WarpWing**
A small clean update to the workflow system. Since there isn't much to be done regarding CodeQL, I simply just deleted some of the comments for visibility. I also cleaned up the SkyCrypt CI file a bit. It should now show the npm ci and npm build phases instead of just plain hiding it.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@ jobs:
       uses: supercharge/mongodb-github-action@1.3.0
       with:
         mongodb-version: ${{ matrix.mongodb-version }}
-    
-    - run: npm ci
```
Comment (nstringham) [inline .github/workflows/node.js.yml:34]: this is invalid yml see nstringham@71fb6e2

**Review by nstringham — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -30,9 +30,9 @@ jobs:
...
-    - run: npm run build --if-present
+        
+    - name: Run NPM Build Process
+      run: npm ci
```
Comment (nstringham) [inline .github/workflows/node.js.yml:35]: this is invalid yml see nstringham@71fb6e2

**Review by nstringham — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (nstringham) [reply .github/workflows/node.js.yml:35]: this will also be resolved by merging #494

**Comment by WarpWing** [comment]
I'm going to cancel this PR and wait for Nate's fix.

**Comment by nstringham** [comment]
> I'm going to cancel this PR and wait for Nate's fix.

Is it ok to merge #494 wich includes #492?

**Comment by WarpWing** [comment]
> > I'm going to cancel this PR and wait for Nate's fix.
> 
> Is it ok to merge #494 wich includes #492?

In theory yes

**Pull request closed without merging**